servicetalk-http-utils/src/main/java/io/servicetalk/http/utils/CacheConnectionHttpLoadBalanceFactory.java

```diff
@@ -191,6 +201,15 @@ private void decrementRefCnt() {
                         }
                     });
         }
+
+        @Override
+        public Result tryRequest() {
+            Result result = delegate().tryRequest();
```

Comment:
is the delegate the controller? (so to make sure the override does not change the behavior)

Reply:
Delegate should be `final FilterableStreamingHttpLoadBalancedConnection delegate` from line 150. The `super.tryRequest()` is from `DelegatingFilterableStreamingHttpLoadBalancedConnection` which is implemented as `return delegate().tryRequest()`.
As an aside, my personal preference is to use `super.tryRequest()` because it's unambiguous but for whatever reason we always use `delegate()` in this code base.

Reply:
ah ok thanks for the clarification!